lib/tasks/on_first_instance.rake

```diff
@@ -0,0 +1,7 @@
+namespace :cf do
+  desc "Only run on the first application instance"
+  task :on_first_instance do
+    instance_index = JSON.parse(ENV["VCAP_APPLICATION"])["instance_index"] rescue nil
+    exit(0) unless instance_index == 0
+  end
+end
```

Comment:
I'm usually not a big fan of code comments, but it might make sense to link to the CF docs that explain why this is needed in this Rake task somewhere ...because looking at it now, it's not super clear what this does